src/anubhav/assignment2/org/Assignment2.java

```diff
@@ -0,0 +1,58 @@
+package anubhav.assignment2.org;
+
+import java.io.File;
+//use to take inputs
+import java.util.Scanner;
+
+
+//declared main()
+public class Assignment2 {
+    private void searchFile()
+    {
+
+        // address of files or folder where to search
+        File file = new File("/home/anubhav");
+        //string to store the name of the file
+        String str ;
+        //  Scanner is declare
+        Scanner in = new Scanner(System.in);
+        // do while is used to run the program again and again
+        do{
+            int flag =0;
+            System.out.println("ENTER THE NAME OF FILE OR FOLDER:\t");
+            //take input
+            str =in.nextLine();
+            //list of file is stored in string
+            String[] files = file.list();
+            //continuous check each file or folder
+            for (String string : files){
+
+                //check the name of file present or not and store in boolean str1
+                Boolean str1 = str.equals(string);
```

Comment:
If it's a boolean variable give something called flag or result , isExists variable name

Reply:
I have changed this approach of searching of file updated in the recent branch.